src/modules/common/config/passportConfig.ts

```diff
@@ -14,12 +14,13 @@ passport.use(
     {
       clientID: GOOGLE_CLIENT_ID,
       clientSecret: GOOGLE_CLIENT_SECRET,
-      callbackURL: process.env.GOOGLE_CALLBACK_URL_DEV,
+      callbackURL:
+        process.env.NODE_ENV !== "production"
```

Comment:
This does not really matter.  when fixed well, should work for both prod and dev